### ISSUE TITLE
error: Error: Unable to resolve module ./hostnameServerApi

### ISSUE THREAD

**Issue by esmizan** [sanengineer/ecommerce-with-react-native#1]
error: Error: Unable to resolve module ./hostnameServerApi from ..\ecommerce-with-react-native\src\services\userService.js:

**Comment by ayyam** [comment]
Same issue for me, error screenshot below,

![image](https://user-images.githubusercontent.com/33388963/160598034-34917a83-8c6d-4ef9-899a-6fc3fc0678d5.png)

**Comment by esmizan** [comment]
Yes. I got so which shown in screenshot. Got missing 'hostnameServerApi'. Thank you!
